modules/microshift-deploying-operators.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * microshift_operators/microshift-deploying-operators.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-deploying-operators_{context}"]
+= Deploying operators using manifests
+
+You can use manually managed operators with {product-title} to create applications that monitor the running services in the cluster. Operators are designed specifically for your applications. These customized pieces of software running inside your cluster can be used implement and automate the common Day 1 operations such as installation and configuration as well as Day 2 operations such as creating backups.
```

Comment:
```suggestion
You can use operators with {product-title} to create applications that monitor the running services in the cluster. Operators are designed specifically for your applications. These customized pieces of software running inside your cluster can be used implement and automate the common operations such as ... .
```

I don't think we care as much about day 1 vs day 2 on MicroShift as we do on OCP, so we don't need separate examples. The actual examples given here won't apply for MicroShift, so maybe we can use a generic phrase where I commented above and list specific examples here?

I removed the "manually managed" phrase because the operators can still be automated, just not via OLM. @DanielFroehlich , maybe you have ideas for alternative phrasing or a way to emphasize that operators work, but differently?